uber/custom_tags.py

```diff
@@ -49,11 +57,15 @@ def event_dates():
 
 @JinjaEnv.jinja_export
 def hour_day_local(dt):
+    if not dt:
+        return ''
     return hour_day_format(dt)
 
 
 @JinjaEnv.jinja_filter
 def timestamp(dt):
+    if not dt:
```

Comment:
I had the exact same thought, but there's no nice way to refactor this. The best we could do is create a decorator that checks for a parameter named `dt`, and returns an empty string if the `dt` argument is empty.

I suppose it's doable, but I'm not certain it is more immediately obvious to a new developer, or if it saves any lines of code :(

In my next commit, I took your advice from above and used the ternary `if` `else` and promoted the empty condition to the front, which is probably the cleanest way to do it.